DependencyInjection/HWIOAuthExtension.php

```diff
@@ -39,15 +41,16 @@ final class HWIOAuthExtension extends Extension
      * @throws InvalidConfigurationException
      * @throws BadMethodCallException
      * @throws InvalidArgumentException
-     * @throws \Symfony\Component\DependencyInjection\Exception\OutOfBoundsException
-     * @throws \Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException
+     * @throws OutOfBoundsException
+     * @throws ServiceNotFoundException
      */
     public function load(array $configs, ContainerBuilder $container)
     {
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config/'));
         $loader->load('controller.xml');
         $loader->load('http_client.xml');
         $loader->load('oauth.xml');
+        $loader->load('resource_owners.xml');
```

Comment:
Split was required due too issue in Symfony 4.4 not allowing usage of `parent` & `instanceof` in same config file.

Reply:
I removed the `instanceof` in later changes, but that split makes sense IMHO as it make it more readable.